packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -197,6 +200,10 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
         const descriptionLocaleKey = `preferencesManagement.${id}.description`;
         // Ensure not to display fallback text as description as its expected that some items might not have its own description
         const description = hasDescription && this._localiseText(descriptionLocaleKey);
+        const requiredToggleAllKeys = ['functional', 'personalized', 'analytical'];
```

Comment:
@kevinrodrigues I have tested this in Aperture NextJS and it seems like if they are not all true, none of them are enabled. I do not want to hold up this PR but I am concerned this is not correct behaviour.